packages/driver/cypress/plugins/server.js

```diff
@@ -43,6 +43,14 @@ const createApp = (port) => {
     })
   })
 
+  app.get('/redirect-timeout', (req, res) => {
+    return Promise
+    .delay(100)
+    .then(() => {
+      return res.send('<html><body>timeout</body></html>')
+    })
+  })
+
```

Comment:
nit: is there something blocking you from creating a `cy.intercept()` instead of a new route in `server.js`? This file be balloonin'. I guess you'd have to update the test to deal with the add'l logs.

Reply:
Yeah, this was my attempt to be minimally invasive.